docs/best-practices/migrating-from-mongodb.rst

```diff
@@ -74,10 +76,10 @@ enable great full-text search capabilities. See `Indices and Fulltext Search`_
 for more information.
 
 CrateDB is able to dynamically extend the schema, so it is not necessary to
-define all columns up-front.
+define all columns up front.
 
 Given the table above, it is possible to insert new columns at the top level of
-the table and insert arbitrary objects into the obj column::
+the table and insert arbitrary objects into the **obj** column::
```

Comment:
```suggestion
the table and insert arbitrary objects into the ``obj`` column::
```

Reply:
I thought this too, but looking at the rest of the files, they have systematically given table elements bold type rather than ``. So that seems to fall under the consistency rule.

Reply:
weird. haha. but sure, okay!